spec/urbanopt/urbanopt_scenario_reports_spec.rb

```diff
@@ -149,14 +149,13 @@
 
 
   it 'can merge end uses' do
-    existing_end_use = URBANopt::Scenario::DefaultReports::ReportingPeriod.new({end_uses: { electricity: { heating: 1, cooling:1 }, natural_gas:{fans: 1, pumps: 1}}})
-    new_end_use = URBANopt::Scenario::DefaultReports::ReportingPeriod.new({end_uses: { electricity: { heating: 1, cooling:1 }, natural_gas:{fans: 1, pumps: 1}}})
+    existing_end_uses = EndUses.new( { electricity: { heating: 1, cooling:1 }, natural_gas:{fans: 1, pumps: 1}} )
```

Comment:
@rawadelkontar hashes are not interchangeable with classes (classes have custom methods, hashes do not), you need to construct the class